lib/process-services-cloud/src/lib/process/process-list/datatable/process-list-datatable-adapter.spec.ts

```diff
@@ -50,4 +50,9 @@ describe('ProcessListDatatableAdapter', () => {
 
         expect(adapter.getColumnType(row, column)).toBe('number');
     });
+
+    it('should initialize constructor with server sortingMode', () => {
+        const adapter = new ProcessListDatatableAdapter([], []);
+        expect(adapter['_sortingMode']).toBe('server');
```

Comment:
the test should not be touching private methods... I suggest making a public accessor to the sorting mode (indexed access to private properties is configured a hack or the temp limitations of TS)

Reply:
Yeah, I'm not a fan of accessing private variables either but didn't want to create a public field just for the sole purpose of testing. I created tests to verify the sort direction of the data but that triggered duplicate code warnings in the PR.

I can expose the field publicly if that's the suggested patter for this type of scenario?

Reply:
the way you access private is more of a hack, any time MS fixes the TS and everything may start failing... or it will me marked as security issue... it's better either not touching the private, or testing other code that involves this private, or making an accessor